### PR TITLE
Update submissions-admin.md

### PR DESCRIPTION
**Pull request by madalenamedo**
The images are missing please contact me and I can provide them alias madalenamedo.
Thank you!

### CODE REVIEW

**Comment by madalenamedo** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by madalenamedo** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 1]
```diff
@@ -183,6 +183,30 @@ Watch this short video to learn how to use admin submissions in Microsoft Defend
...
+
+4. On the **Add new submission** flyout that appears, enter the following information:
+
+   - Click **Browse files** to find and select the file to submit.
+
+   - **Select the submission type**: You can choose the value **Files** or **File hash**.
+
+   - **This file should have been categorized as**: Select **Malware** or **Unwanted Software**.
+
+   - **Choose the priority**: Select **Low - bulk file or file hash submission** or **Medium - standard submission** or **High - need immediate attention (3 allowed per org per day)**. If you're not sure, use your best judgment. This option is only availabel if you choose the option **Files** in **Select the submission type**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
   - **Choose the priority**: Select **Low - bulk file or file hash submission** or **Medium - standard submission** or **High - need immediate attention (3 allowed per org per day)**. If you're not sure, use your best judgment. This option is only available if you choose the option **Files** in **Select the submission type**.
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 2]
```diff
@@ -305,6 +329,30 @@ After a few moments, an allow entry will appear on the **URL** tab on the **Tena
...
+
+4. On the **Add new submission** flyout that appears, enter the following information:
+
+   - Click **Browse files** to find and select the file to submit.
+
+   - **Select the submission type**: You can choose the value **Files** or **File hash**.
+
+   - **This file should have been categorized as**: Verify the value **Clean** is selected.
+
+   - **Choose the priority**: Select **Low - bulk file or file hash submission** or **Medium - standard submission** or **High - need immediate attention (3 allowed per org per day)**. If you're not sure, use your best judgment. This option is only availabel if you choose the option **Files** in **Select the submission type**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
   - **Choose the priority**: Select **Low - bulk file or file hash submission** or **Medium - standard submission** or **High - need immediate attention (3 allowed per org per day)**. If you're not sure, use your best judgment. This option is only available if you choose the option **Files** in **Select the submission type**.
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 3]
```diff
@@ -462,6 +510,46 @@ After a few moments, an allow entry will appear on the **URL** tab on the **Tena
...
+     - **Submission name**<sup>\*</sup>
+     - **Submission ID**<sup>\*</sup>
+     - **Submitted by**
+     - **Date submitted**<sup>\*</sup>
+     - **Submission Type**
+     - **Reason for submitting**<sup>\*</sup>
+     - **Status**<sup>\*</sup>
+     - **Priority**<sup>\*</sup>
+     - **Customer comment**
+     - **Researchedr comment**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
     - **Researcher comment**
```

Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md:531]:
```suggestion
     - **Researcher comment**
```

File: microsoft-365/security/office-365-security/submissions-admin.md [hunk 4]
```diff
@@ -462,6 +510,46 @@ After a few moments, an allow entry will appear on the **URL** tab on the **Tena
...
+
+     When you're finished, click **Apply**.
+
+   - To filter the entries, click ![Filter icon.](../../media/m365-cc-sc-filter-icon.png) **Filter**. The following values are available in the **Filter** flyout that appears:
+     - **Date submitted**: **Start date** and **End date** values.
+     - **Submitted as**: The values **Unknown**, **Clean**, **False positive**, **Experimental false positive**, **Malware**, **Spyware**, **Unwanted Software**, **Pua false positive**, and **Night watch unknown**.
+     - **Status**: The values **New**, **Unassigned**, **Assigned**, **Pending**, **Resolved**, **Closed**, **Downloading**, **Sample collection**, **Sample collection failure**, **Rejected**, and **Review timed out**.
+     - **Submission ID**: A GUID value that's assigned to every submission.
+     - **Priority**: The values **Low**, **Medium** or **High**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/submissions-admin.md]:
```suggestion
     - **Priority**: The values **Low**, **Medium**, or **High**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @madalenamedo, kindly commit the suggested changes. Many thanks.

**Comment by madalenamedo** [comment]
@JohanFreelancer9 done, thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by denisebmsft** [comment]
Images requested. Will review and merge once they're in.

**Pull request merged**